Anlab.Mvc/Views/Prices/Index.cshtml

```diff
@@ -0,0 +1,60 @@
+@using Humanizer
+@model Anlab.Core.Models.TestItemModel[]
+
+@{
+    ViewBag.Title = "Current Prices";
+}
+
+<div class="col">
+    <table class="table">
+        <thead>
+        <tr>
+            <th>@Html.DisplayNameFor(model => model[0].Category)</th>
+            <th>@Html.DisplayNameFor(model => model[0].Group)</th>
+            
+            <th>@Html.DisplayNameFor(model => model[0].Analysis)</th>
+            @*<th>@Html.DisplayNameFor(model => model[0].Id)</th>*@
+            <th>@Html.DisplayNameFor(model => model[0].InternalCost).Humanize()</th>
+            <th>@Html.DisplayNameFor(model => model[0].InternalSetupCost).Humanize()</th>
+            <th>@Html.DisplayNameFor(model => model[0].ExternalCost).Humanize()</th>
+            <th>@Html.DisplayNameFor(model => model[0].ExternalSetupCost).Humanize()</th>

```

Comment:
@jSylvestre Change the header values for the last 4 columns to represent UC/non-UC rates.  Use a colspan to span a major grouping, like so

<thead><tr><th colspan="3"></th><th colspan="2">UC Clients</th><th colspan="2">Non-UC Clients</th></tr><tr>[...your original tr except drop internal/external]</tr></thead>

That should give it a nice tiered look and also group by what they understand, assuming datatables is ok with that.  If it isn't, just change the name to UC Cost instead of Internal Cost, etc, etc.

Reply:
@srkirkland Like this?

![image](https://user-images.githubusercontent.com/469010/32518093-6f093d3a-c3bd-11e7-8f90-6ddf2325927c.png)